src/cmd/manage_nsfs.js

```diff
@@ -61,6 +61,7 @@ Bucket Options:
     # required for add, replace 
     --email <email>             (default none)          Set the email for account/bucket.
     --path <dir>                (default none)          Set the bucket path.
+    --creation_date <time>      (default current time)          Set time bucket creation. Formate YYYY-MM-DDTHH:mm:ssZ,
```

Comment:
can you please update the bucket schema of the nsfs standalone doc with this, also I wonder if we want to have also creation_date on account as well, WDYT?

Reply:
so the `creation_date` will be current date, right?

Reply:
yeah, sounds good to me